magefile.go

```diff
@@ -8,31 +8,159 @@
 package main
 
 import (
+	"context"
+	"fmt"
+	"os"
 	"path/filepath"
+	"runtime"
+	"strconv"
+	"strings"
 
-	//mage:import
 	"github.com/elastic/elastic-agent-libs/dev-tools/mage"
+	devtools "github.com/elastic/elastic-agent-shipper/dev-tools/common"
+	"github.com/elastic/elastic-agent-shipper/tools"
+	"github.com/pkg/errors"
+
+	//mage:import
+
 	"github.com/elastic/elastic-agent-libs/dev-tools/mage/gotool"
 	"github.com/magefile/mage/mg"
 	"github.com/magefile/mage/sh"
 )
 
-var (
-	goLicenserRepo = "github.com/elastic/go-licenser@v0.4.1"
+const (
+	GoreleaserRepo = "github.com/goreleaser/goreleaser@v1.6.3"
```

Comment:
please ignore this if you don't like it, but I would separate `.1.6.3` to a variable ending with version so in case I forgot where it is defined I can guess based on full text search